modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```diff
@@ -512,179 +549,179 @@
       "filename_template" : "ftp://ftp.ebi.ac.uk/pub/databases/eva/PRJEB22988/Svardal_et_al_2017_vervet_monkey_SNPs_imputed_phased.vcf.gz"
     },
     {
-      "id": "100_mammals.gerp_conservation_score",
+      "id": "103_mammals.gerp_conservation_score",
       "species": "bos_taurus",
       "assembly": "ARS-UCD1.2",
       "type": "remote",
-      "filename_template" : "ftp://ftp.ensembl.org/pub/release-99/compara/conservation_scores/100_mammals.gerp_conservation_score/gerp_conservation_scores.bos_taurus.ARS-UCD1.2.bw",
+      "filename_template" : "ftp://ftp.ensembl.org/pub/release-99/compara/conservation_scores/103_mammals.gerp_conservation_score/gerp_conservation_scores.bos_taurus.ARS-UCD1.2.bw",
```

Comment:
Should the directory structure be release-100 for the gerp files?

Reply:
fixed

Reply:
will be fixed soon. I need to fix a merge conflict first